kvstore/sql.go

```diff
@@ -217,19 +243,29 @@ func (o SQL) Set(key string, val string) error {
 
 	defer func() { _ = txn.Rollback() }()
 
-	// nolint:gosec
-	// o.TableName has been checked by isSafeTblName on init
-	delQ := fmt.Sprintf("DELETE FROM %s WHERE key = ?", o.TableName)
+	delQuery := sq.Delete(o.TableName).
+			Where(sq.Eq{"kv_key": key}).
+			PlaceholderFormat(o.Placeholder)
+
+	query_text, args, err := delQuery.ToSql()
+	if err != nil {
+		panic(err)
```

Comment:
ditto (`panic`ing in a package)

Reply:
same reason as above -- this is a run-time check for a compile-time error.